app/src/main/java/org/mozilla/fenix/collections/CollectionCreationUIView.kt

```diff
@@ -104,6 +104,7 @@ class CollectionCreationUIView(
 
         view.tab_list.run {
             adapter = collectionCreationTabListAdapter
+            itemAnimator = null
```

Comment:
I'm not sure I understand why this helps.

Reply:
This was for the flickering of the tabs list as it's binding. Since this list never updates while a user is looking at it, we won't need to animate it and this stops the flickering.